test/libsolidity/syntaxTests/nameAndTypeResolution/145_external_base_visibility.sol

```diff
@@ -5,4 +5,4 @@ contract derived is base {
     function g() public { base.f(); }
 }
 // ----
-// TypeError: (100-106): Member "f" not found or not visible after argument-dependent lookup in type(contract base).
+// TypeError: (100-108): Cannot call function via contract name.
```

Comment:
Is this an error message we had previously?

Reply:
It's the one I added for https://github.com/ethereum/solidity/pull/8105 .
And I'd say it fits much better here than the old one.